policyengine_us/parameters/gov/states/ny/tax/income/credits/real_property_tax/maximum/elderly.yaml

```diff
@@ -78,7 +78,9 @@ metadata:
   threshold_unit: currency-USD
   threshold_period: year
   amount_unit: currency-USD
-  label: NY real property tax credit elderly maximum amount
+  label: New York real property tax credit elderly maximum amount
   reference:
     - title: Section 606 Credits against tax (e)
       href: https://www.nysenate.gov/legislation/laws/TAX/606
+    - title: New York Instructions for Form IT-201 Line 67
+      href: https://www.tax.ny.gov/pdf/2023/printable-pdfs/inc/it201i-2023.pdf#page=26
```

Comment:
wrong page here as well, I think we used page 26 a lot

Reply:
I went through all parameters again and added specific parameter references when available. However, some didn't have them.

Reply:
which ones?

Reply:
> which ones?

couldn't find any of the CDCC parameters,
supplemental_match for EITC,
applicable_percentage for College Tuition,
max_rent for Real Property Tax,